DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: iSEE
 Title: Interactive SummarizedExperiment Explorer
-Version: 1.2.1
-Date: 2018-10-24
+Version: 1.2.2
+Date: 2018-02-26
```

Comment:
2019?

Reply:
Ah, but @kevinrue is a time traveller.(_Yes, iSEE is bigger on the inside!_)

Incidentally, this is why running `R CMD check` locally is helpful, because you get a warning about the date being... out of date. CI just gives a pass/fail and no one checks the warnings.

Reply:
Whoops. Fixed!